src/lib/Scenes/City/City.tsx

```diff
@@ -102,8 +102,10 @@ export class CityView extends Component<Props, State> {
               />
               <ScrollView
                 contentInset={{ bottom: bottomInset }}
-                onLayout={layout => (this.scrollViewVerticalStart = layout.nativeEvent.layout.y)}
-                scrollEnabled={isDrawerOpen}
+                onLayout={layout => {
+                  this.scrollViewVerticalStart = layout.nativeEvent.layout.y
+                  NativeModules.ARNotificationsManager.postNotificationName("ARLocalDiscoveryCityGotScrollView", {})
```

Comment:
I wonder if this could get called too often and if moving this to the `ref` callback would work too?

Reply:
I initially tried the `ref` callback but it's invoked before the scroll view has been added to the UIView hierarchy. I'll profile to see how often this gets called – my impression from the docs was that it would be fine, but let's verify.

Reply:
Yup, confirmed that this only gets called when switching tabs. I think that's acceptable.

Reply:
Ah gotcha